src/main/scala/rocketchip/BaseTop.scala

```diff
@@ -67,7 +67,8 @@ abstract class BaseTopModule[+L <: BaseTop, +B <: BaseTopBundle](
   val io: B = b
 
   val coreplex = p(BuildCoreplex)(outer.c, p)
-  val coreplexIO = coreplex.io
+  val coreplexIO = Wire(new BaseCoreplexBundle(outer.c)(p){})
```

Comment:
Yeah, I thought of doing something like that bus wasn't sure how. I'll try it, thanks.